src/controllers/albumController.js

```diff
@@ -4,17 +4,48 @@ const CustomError = require('../errors');
 const logger = require('../../logs/logger');
 
 const createAlbum = async (req, res) => {
+  /*
+     #swagger.summary = 'Create new album and save it to the database'
+    
+     #swagger.responses[201] = {
+				description: 'Albums was successfully created.',
+        schema: [{ $ref: '#/definitions/Album'' }]
+		 }
+		 
+  */
   req.body.user = req.user.userId;
   const album = await Album.create(req.body);
   res.status(StatusCodes.CREATED).json({ album });
 };
 
+// question for Akos - what does minimin means on line 20?
 const getAllAlbums = async (req, res) => {
-  const albums = await Album.find({});
+  /*
+     #swagger.summary = 'Fetch all albums in a database'
+    
+     #swagger.responses[200] = {
+				description: 'Albums  successfully fetched.',
+        schema: { albums: [{ $ref: '#/definitions/Album' }] }
```

Comment:
According to the return value of this function we're returning an object with two fields: `'albums'` which has a list of albums, and `'count'` which is a number

```suggestion
        schema: { albums: [{ $ref: '#/definitions/Album' }], count: 1 }
```